snmp/datadog_checks/snmp/data/default_profiles/_dell-rac.yaml

```diff
@@ -205,6 +230,30 @@ metrics:
           OID: 1.3.6.1.4.1.674.10892.5.4.300.70.1.8
           name: intrusionLocationName
         tag: intrusion_location_name
+  - MIB: IDRAC-MIB-SMIv2
+    table:
+      OID: 1.3.6.1.4.1.674.10892.5.4.600.12
+      name: powerSupplyTable
+    symbols:
+      - OID: 1.3.6.1.4.1.674.10892.5.4.600.12.1.6
+        name: powerSupplyOutputWatts
+      - OID: 1.3.6.1.4.1.674.10892.5.4.600.12.1.9
+        name: powerSupplyMaximumInputVoltage
+      - OID: 1.3.6.1.4.1.674.10892.5.4.600.12.1.16
+        name: powerSupplyCurrentInputVoltage
+    metric_tags:
+      - column:
+          OID: 1.3.6.1.4.1.674.10892.5.4.600.12.1.1
+          name: powerSupplychassisIndex
+        tag: chassis_index
```

Comment:
1/ Any reason for using `chassis_index` instead of `power_supplychassis_index` ?

2/ same question for `device_fqdd` instead of `power_supply_fqdd` ?

Reply:
Yes, if you look at the other metrics in this MIB, all of the chassisIndex'es are tagged as just `chassis_index`, since the descriptions of them are identical: `"This attribute defines the index (one based) of the associated system chassis."`.  There are some exceptions (for example with `networkDeviceChassisIndex`, but I decided not to touch since it was done a long time ago). 

I agree with `device_fqdd`, the descriptions are different in all cases so I'll change that.

Reply:
Makes sense if there are existing tags names like that.